endercore/src/main/resources/META-INF/mods.toml

```
@@ -1,34 +1,27 @@
-modLoader="javafml" #mandatory
-loaderVersion="[37,)" #mandatory This is typically bumped every Minecraft version by Forge. See our download page for lists of versions.
-issueTrackerURL="http://my.issue.tracker/" #optional
+modLoader="javafml"
+loaderVersion="[39,)"
+#issueTrackerURL="http://my.issue.tracker/" #TODO: Endercore REPO
 license="CC0"
 
-[[mods]] #mandatory
-modId="endercore" #mandatory
-version="${file.jarVersion}" #mandatory
-displayName="EnderCore" #mandatory
-#updateJSONURL="http://ci.tterrag.com/job/EnderCore" #optional
-displayURL="http://enderio.com" #optional
-logoFile="examplemod.png" #optional
-credits="" #optional
-authors="tterrag, CrazyPants, HenryLoenwind, Rover656, HypherionSA" #optional
+[[mods]]
+modId="endercore"
+version="${file.jarVersion}"
+displayName="EnderCore"
+#updateJSONURL="http://ci.tterrag.com/job/EnderCore"
+displayURL="http://enderio.com"
+logoFile="examplemod.png"
+credits=""
+authors="tterrag, CrazyPants, HenryLoenwind, Rover656, HypherionSA, Ferri_Arnus"
```

you missed one again
```suggestion
authors="tterrag, CrazyPants, HenryLoenwind, Rover656, HypherionSA, Ferri_Arnus, agnor99"
```